src/main/java/io/github/nucleuspowered/nucleus/modules/ban/commands/TempBanCommand.java

```diff
@@ -34,10 +40,27 @@
 @NoCost
 public class TempBanCommand extends CommandBase<CommandSource> {
 
+    @Inject private BanConfigAdapter bca;
     private final String user = "user";
     private final String reason = "reason";
     private final String duration = "duration";
 
+    private final String bypassMaximumLength = PermissionRegistry.PERMISSIONS_PREFIX + "tempban.bypass";
```

Comment:
Yes, it's an exemption from the cap.
On Fri, 29 Jul 2016 at 19:42, Oblivion <notifications@github.com> wrote:

> In
> src/main/java/io/github/nucleuspowered/nucleus/modules/ban/commands/TempBanCommand.java
> <https://github.com/NucleusPowered/Nucleus/pull/263#discussion_r72841568>:
>
> >      private final String user = "user";
> >      private final String reason = "reason";
> >      private final String duration = "duration";
> >
> > +    private final String bypassMaximumLength = PermissionRegistry.PERMISSIONS_PREFIX + "tempban.bypass";
>
> Is this an exemption? The permission node is used to bypass the maximum
> time when temporarily banning another user.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/NucleusPowered/Nucleus/pull/263/files/bb4fd4607a005ae29bddf3d828ddd2abc42a9ac4#r72841568>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB0OJ4O6DvUHOl058eskKrAXEDErr-pfks5qakmGgaJpZM4JOnE0>
> .
>